pkg/install/install_test.go

```diff
@@ -120,3 +123,62 @@ func TestDeployARMTemplate(t *testing.T) {
 		})
 	}
 }
+
+func TestAddResourceProviderVersion(t *testing.T) {
+
```

Comment:
need to kill this empty line. we don't start functions with an empty line..

Reply:
gotcha